packages/harpparts/src/pitch/types/types.ts

```diff
@@ -32,11 +34,15 @@ export type UnnaturalDisplayValues = {
 }
 
 export type NaturalPitch = {
+  readonly type: HarpPartTypes.Pitch
   readonly id: PitchIds
   readonly contextualDisplayValues: NaturalDisplayValue
+  readonly simpleSplitValue: [string, string]
```

Comment:
Raise issue to investigate and unify the simpleSplitValue and contextualDisplayValues uses if possible.

Reply:
Raise https://github.com/js-jslog/harpguru/issues/74